### PR TITLE
Create a Mode Rules box

### LINKED ISSUE #20: UI Component | Rules for each game mode
old hash: #11
Acceptance Criteria:
Verify or consult before submitting the task!
- [x] Component defined in a separate *.js file,
- [x] Component accepts text to display,
- [x] Component displays the text,
- [x] Component displays its header "Mode Rules".
- [x] Component displays an icon on the left side of its header
 - [ ] Component has been tested by unit tests
 - [x] Component appearance according to the project

https://bit.ly/3s4Zymw

### PR DESCRIPTION
**Pull request by synowa**
closes #20  fixex #20
Screenshot 2021-01-10 at 20 35 44

### CODE REVIEW

File: src/app/components/ModeRules.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const ModeRules = (mode) => {
+  const rulesElement = document.createElement('div');
+  rulesElement.classList.add('rules');
+  const rulesHeader = document.createElement('header');
+  rulesHeader.classList.add('rules-header');
+  const rulesIcon = document.createElement('span');
+  rulesIcon.classList.add('rules-icon', 'fa', 'fa-graduation-cap');
+  const rulesTextElement = document.createElement('div');
+  rulesTextElement.classList.add('rules-text');
```
Comment (SebastianBabinski1) [inline src/app/components/ModeRules.js:18]: Paulina said that we should declarate all variables first, then methods like classList.add

File: styles/components/_modeRules.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.rules {
+  padding: 1.25rem;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```
Comment (SebastianBabinski1) [inline styles/components/_modeRules.scss:3]: We must declare variable of this box-shadow in _variables.scss, because I see this value often.

**Review by SebastianBabinski1 — COMMENTED** [review]
Generally looks good, please make minor corrections included in comments

File: src/app/components/ModeRules.js (hunk 1 above)
Comment (synowa) [reply src/app/components/ModeRules.js:18]: Sure, I'll change that.

File: styles/components/_modeRules.scss (hunk 2 above)
Comment (synowa) [reply styles/components/_modeRules.scss:3]: I'll change that in another PR together with some other fixes.

**Review by Enessetere — APPROVED** [review]
Looks good to me, works as designed ;)

**Review by SebastianBabinski1 — APPROVED** [review]
Good job :)

**Pull request merged**